packages/miniflare/src/plugins/kv/index.ts

```diff
@@ -41,19 +38,25 @@ export const KVSharedOptionsSchema = z.object({
 });
 
 const SERVICE_NAMESPACE_PREFIX = `${KV_PLUGIN_NAME}:ns`;
-const KV_STORAGE_SERVICE_NAME = `${KV_PLUGIN_NAME}:storage`;
 const KV_NAMESPACE_OBJECT_CLASS_NAME = "KVNamespaceObject";
-const KV_NAMESPACE_OBJECT: Worker_Binding_DurableObjectNamespaceDesignator = {
-	serviceName: SERVICE_NAMESPACE_PREFIX,
-	className: KV_NAMESPACE_OBJECT_CLASS_NAME,
-};
 
 function isWorkersSitesEnabled(
 	options: z.infer<typeof KVOptionsSchema>
 ): options is SitesOptions {
 	return options.sitePath !== undefined;
 }
 
+function addPersistPathSuffix(
+	baseName: string,
+	persistPath: Persistence
+): string {
+	if (typeof persistPath === "string") {
+		return `${baseName}:${persistPath}`;
```

Comment:
Is there any reserved characters for service name?

Reply:
url encode